app/assets/stylesheets/custom.scss

```diff
@@ -26,6 +42,55 @@ textarea {
   }
 }
 
+body {
+  padding-top: 60px;
+}
+
+/* header */
+#logo {
+  float: left;
+  margin-right: 10px;
+  font-size: 1.7em;
+  color: #fff;
```

Comment:
![image](https://user-images.githubusercontent.com/65896888/91674387-a5405300-eb62-11ea-86c0-016d2e3f7009.png)

Reply:
![image](https://user-images.githubusercontent.com/68435978/91675265-4d0f4e00-eb6e-11ea-85f4-3ec44abadff1.png)
I already have a .scss file for color.